mico-core/src/main/java/io/github/ust/mico/core/service/MicoStatusService.java

```diff
@@ -66,10 +89,27 @@ public MicoStatusService(PrometheusConfig prometheusConfig, MicoKubernetesClient
     public MicoApplicationStatusDTO getApplicationStatus(MicoApplication micoApplication) {
         MicoApplicationStatusDTO applicationStatus = new MicoApplicationStatusDTO();
         List<MicoService> micoServices = serviceRepository.findAllByApplication(micoApplication.getShortName(), micoApplication.getVersion());
+        int podCount = 0;
+        int requestedReplicasCount = 0;
+        int availableReplicasCount = 0;
         for (MicoService micoService: micoServices) {
             MicoServiceStatusDTO micoServiceStatus = getServiceStatus(micoService);
+            podCount += micoServiceStatus.getPodInfo().size();
+            requestedReplicasCount += micoServiceStatus.getRequestedReplicas();
+            availableReplicasCount += micoServiceStatus.getAvailableReplicas();
+            // Remove the current application's name to retrieve a list with only the names of other applications that are sharing a service
+            for (MicoUsingApplicationDTO micoUsingApplicationDTO: micoServiceStatus.getUsingApplications()) {
+                if (micoUsingApplicationDTO.getName().equals(micoApplication.getName())) {
```

Comment:
`name` of a *MicoApplication* is not enforced to be unique! Use `shortName` and `version` instead.

Reply:
Change to check for equal **shortName** and **version** instead of **name**.